# Project 1
[TOC]

# Introduction
I studied the EMSY gene, or known fully as EMSY transcriptional repressor, BRCA2 interacting. This gene acts a repressor and is also involved in DNA repair, organizing chromatin, and regulating [transcription](https://rgd.mcw.edu/rgdweb/report/gene/main.html?id=1345109). I chose this specific gene because I read [several](https://www.sciencedirect.com/science/article/pii/S0091674917315749#sec1) [studies](https://www.sciencedirect.com/science/article/pii/S1323893019301595) that show a link between EMSY and different types of allergies, namely atopic dermatitis and peanut/food allergies. One study shows that the EMSY gene is risk locus for peanut allergy and other food allergies, while another study found that overexpression of the EMSY gene is a factor in developing atopic dermatitis. One type of data I searched for was gene expression, since the expression levels of this gene play a major factor in whether or not a person suffers from the allergies mentioned above. Another type of data I searched for was common mutations in the gene’s sequence, to see how and which variations affected the gene’s expression.



# Database 1: gnomAD
## General Information
![](https://d2908q01vomqb2.cloudfront.net/c5b76da3e608d34edb07244cd9b875ee86906328/2020/09/29/gnomAD-logo-300x168.jpg)
*source: https://d2908q01vomqb2.cloudfront.net/c5b76da3e608d34edb07244cd9b875ee86906328/2020/09/29/gnomAD-logo-300x168.jpg
The Genome Aggregation Database, also known as gnomAD, contains genome and exome sequencing data. Biological questions this database could address are: What types of variation are possible for gene “X”?, If gene “X” has certain mutations, what are the consequences for the rest of the gene?, and What does the transcript for gene “X” look like? This database has secondary characteristics and is a collection of exome and genome data sourced from 60+ different sequencing projects. A majority of the data was processed by the Broad Institute. Heidi Rehm and Mark Daly serve as co directors and manage the project, along with a number of other individuals on the steering committee. Another sign that it is a secondary database is the fact that it references other databases such as: Ensembl and UCSC Browser.

## Data Usage and Accessibility
In terms of how the data is stored, the data is depicted visually with one form being a graph showing coverage of transcripts, and another being an interactive slider where variants can be selected/unselected. Variant data is also displayed using a table. The graphical data can be directly downloaded as an .sgv file. The data on variants can be directly downloaded as a .csv file. There are no restrictions on how one is allowed to use the data and it can be used without giving attribution to the database. However, the database does encourage people to acknowledge it, give attribution, and provide links to sources of information. 

## Article Referencing the Database
This database was cited in an [article](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6683598/) titled: “EMSY expression affects multiple components of the skin barrier with relevance to atopic dermatitis”. The study was looking at how the expression level of EMSY affects the formation and strength of the skin barrier. This study used this database in order to look at loss of function mutations for EMSY, and how frequent those mutations are. 

## Gene Entry and Takeaways
There is one entry for this gene in this database. In the [entry](https://gnomad.broadinstitute.org/gene/ENSG00000158636?dataset=gnomad_r2_1), there are two main parts. The first part depicts transcripts, and their depths of coverage shown on a graph. The second part shows a slider, where one can look at different variants along the gene. As the slider moves, the table below changes to reflect what variants are in the area the slider is currently in. From this database, I learned that there are over 1000 possible variants within this gene, which really surprised me.

# Database 2: Rat Genome Database
## General Information
![](https://rgd.mcw.edu/rgdweb//common/images/rgd_logo.jpg)
*source: https://rgd.mcw.edu/rgdweb//common/images/rgd_logo.jpg
The Rat Genome Database, or RGD,  contains disease related, genetic, genomic, and phenotypic data pertaining to rats, mice, humans, chinchillas, bonobos, dogs, 13-inch ground squirrels and pigs. Types of biological questions this database could address include: How do certain chemicals affect gene “X”?, How does gene “X” function similarly/differently in humans vs. rats?, and What possible disease(s) is linked to gene “X”?This database has both primary and secondary characteristics. The database states that lots of its data comes directly from  itself, while it also sources additional information from other databases. The data is curated by RGD curators, and the database as a whole is overseen by a principal investigator along with research scientists and curators, and a team of other people who manage other aspects.

## Data Usage and Accessibility
A majority of the data is stored in tables, as well as subcategories with many links to annotations and links to other databases. Jbrowse is available for accessing parts of the data. Other parts of the data can be directly downloaded as a .csv file. The main restriction on usage of data from this database is that one is not allowed to prevent other people from using the data in a way that the license allows, using legal terms or tech measures. Other than that, one is free to share and adapt the material any way they would like to.

## Article Referencing the Database
This database was cited in an [article](https://www.sciencedirect.com/science/article/pii/S0898656818300482) titled: “Identification of macrophage-related candidate genes in lupus nephritis using bioinformatics analysis”.  The researchers from the study used this database mainly for its disease portals, which provides information on the relationship between genes and disease for rats, humans, and mice. They used the disease portals to look at genes related to lupus nephritis, which was their disease of interest.

## Gene Entry and Takeaways
There are 31 entries when searching for the EMSY gene in this database. One [entry](https://rgd.mcw.edu/rgdweb/report/gene/main.html?id=1345109) for humans, displays several different categories of information pertaining to this gene. The top of the page shows general information, such as name, orthologs, position, etc. Below this are different categories, each with sub categories that drop down. The main categories include annotation, genomics, expression, sequence, and additional information. From this database, I learned that EMSY has orthologs in 9 other animals. I also found out that this gene is not only linked to atopic dermatitis, but it is also linked to prostate cancer, breast neoplasms, and polycystic ovary syndrome.

# Comparison
## Types of Data
In terms of database characteristics, the RGD and gnomAD differ in the types of data they hold and differ in how they store and organize data. RGD contains many more types of data than gnomAD. While gnomAD contains data on variants in the gene as well as coverage depth for transcripts, RGD has annotations on disease relation, comparative map data, RNA - Seq expression, nucleotide sequences and more. It is important for a database to have more types of data, because when one is researching a gene of interest, the different types of data will help give a better understanding of how the gene functions and how it affects other genes. It also may expose the person to information that they were not initially looking for, but ends up being important in their research. More data in one database also makes it easier for the user, since they would not have to look up different databases for each type of data. 

## Data Storage and Organization
The Rat Genome Database and Genome Aggregation Database differ in the way they store and organize their data. The gnomAD displays all of its available information at once, with two different visuals and a very large table. RGD meanwhile displays categories of information with subcategories that either link to more information or drop down to show tables or graphs. The way databases store and organize data is important because it affects the user’s experience greatly. I think that the way RGD stores its information is better because it is not as overwhelming as gnomAD and breaks down the different types of data in a way that is easy to understand. The way gnomAD displays data can be very hard to understand, especially if one is not familiar with all of the terms and initialisms used. This characteristic is important for databases because it plays a role in determining whether or not a user will want to revisit the database for information.

